src/commcare_cloud/environment/schemas/terraform.py

```diff
@@ -19,7 +19,7 @@ class TerraformConfig(jsonobject.JsonObject):
     servers = jsonobject.ListProperty(lambda: ServerConfig)
     proxy_servers = jsonobject.ListProperty(lambda: ServerConfig)
     rds_instances = jsonobject.ListProperty(lambda: RdsInstanceConfig)
-    redis = jsonobject.ObjectProperty(lambda: RedisConfig)
+    redis = jsonobject.ObjectProperty(lambda: RedisConfig, default=None)
```

Comment:
why is this becoming optional?

Reply:
This part of `terraform.yml` hooks into a terraform module that creates an elasticache instance, but we ended up _not_ using elasticache, at least not yet. (When migrating our cache data into elasticache ended up not looking super straightforward, I just cut bait and set redis up on a VM like we do elsewhere. We may very well get value eventually out of using elasticache, just didn't seem worth slowing down the AWS migration in general for at the time.)

This schema change basically just says if you're not using elasticache, you can just omit the `redis` section.

Reply:
should it not then be called `elasticache` instead of `redis`?

Reply:
Great point :) I'll make a note to do that in a separate PR